changelog.txt

```diff
@@ -1,6 +1,7 @@
 *** Changelog ***
 
 = 5.x.x - 2021-xx-xx =
+* Tweak - Moved the `WC_Gateway_Stripe::admin_scripts` method to `WC_Stripe_Settings_Controller::admin_scripts`.
```

Comment:
I added a changelog entry because the `WC_Gateway_Stripe::admin_scripts` method was tagged with a `@since` annotation.